assets/src/components/parts/janus-input-number/InputNumber.tsx

```diff
@@ -60,19 +135,26 @@ const InputNumber: React.FC<PartComponentProps<InputNumberModel>> = (props) => {
     if (sValue !== undefined) {
       setInputNumberValue(sValue);
     }
-    const sCssClass = currentStateSnapshot[`stage.${id}.customCssClass`];
-    if (sCssClass !== undefined) {
-      setCssClass(sCssClass);
-    }
+
     //Instead of hardcoding REVIEW, we can make it an global interface and then importa that here.
     if (initResult.context.mode === contexts.REVIEW) {
       setEnabled(false);
     }
-    setReady(true);
-  }, []);
+    setReady((currentValue) => {
+      if (currentValue) {
+        console.warn(
```

Comment:
I think, initialize() getting called multiple times, is the source of the bug where the input will be empty & enabled.

I wasn't able to figure out why that might be happening. Added this console warning so if we see the bug happen, and this message is in the console, it will point to that being the cause.